CommandsManual.md

```diff
@@ -302,6 +309,11 @@ The Auth file is of the form:
         "bitbucketSrvUser": "BITBUCKET_SERVER_USER",
         "bitbucketSrvToken": "BITBUCKET_SERVER_TOKEN",
         "profileType": "bitbucketSrv"
+    },
+    "AZURE_REPOS_USERNAME": {
+        "azureReposUser": "AZURE_REPOS_USER",
```

Comment:
The constant should be consistent between these two lines (both should be `AZURE_REPOS_USERNAME`).

Reply:
Fixed it, thank you for the comment.